### PR TITLE
chore: wallet desktop tutorial

### PR DESCRIPTION
**Pull request by r4mmer**

### CODE REVIEW

File: doc/WALLET_DESKTOP.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+- On the ledger device a prompt to verify the address will appear, you should verify that the address on the screen matches the address on the device.
+- After checking the address you must click with both buttons on the `Verify` screen (on the ledger device)
+
+Any errors on the address will make you loose your tokens, only trust addresses that match the ledger generated address.
+
+## Send HTR Tokens
+
+Go to the "Send Tokens" screen and here you can enter which outputs your transaction is going to have.
+
+![](images/05-send-tokens.png)
```
Comment (pedroferreira1) [inline doc/WALLET_DESKTOP.md:74]: No image description?

**Review by pedroferreira1 — APPROVED** [review]

File: doc/WALLET_DESKTOP.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+# Wallet Desktop
```
Comment (pedroferreira1) [inline doc/WALLET_DESKTOP.md:1]: Maybe we can have another title here (and change the file name as well). To make it clear the tutorial is about using ledger.

**Review by pedroferreira1 — APPROVED** [review]

File: doc/WALLET_DESKTOP.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+## View TX History
+
+The `Wallet` main screen has a list of previous transactions with basic info on the transaction and a link to view transaction info.
+
+![Wallet screen](images/03-wallet-screen.png)
+
+Information on transaction:
+
+- Date and time of the TX
+- Id of the transaction (with link to tx info page)
```
Comment (obiyankenobi) [inline doc/WALLET_DESKTOP.md:41]: Using both TX and transaction in the document. I'd use transaction all around.

File: doc/WALLET_DESKTOP.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+![Tx info screen](images/06-tx-info-screen.png)
+
+By clicking on the TX id on the `View TX History` a page with all information on the transaction opens.
+
+Here you can check any information pertaining to the transaction.
+
+## View Public Address
+
+To receive tokens the sender must input an address to send the tokens.
```
Comment (obiyankenobi) [inline doc/WALLET_DESKTOP.md:57]: I'd remove this sentence, as it seems kind of disconnected to the "View Public Address" section. I imagine people using a Ledger know why addresses are needed.

File: doc/WALLET_DESKTOP.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+![Show address](images/04-show-full-address.png)
+
+Your wallet has many addresses, if you want to get one to receive tokens you need to:
```
Comment (obiyankenobi) [inline doc/WALLET_DESKTOP.md:61]: Split sentences.

File: doc/WALLET_DESKTOP.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+- On the `Wallet` main screen upper right corner there is a box with the current address
+- Click on "Show full address" to show the complete address, check your ledger device
+- On the ledger device a prompt to verify the address will appear, you should verify that the address on the screen matches the address on the device.
```
Comment (obiyankenobi) [inline doc/WALLET_DESKTOP.md:65]: Split sentences.

File: doc/WALLET_DESKTOP.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+- After checking the address you must click with both buttons on the `Verify` screen (on the ledger device)
+
+Any errors on the address will make you loose your tokens, only trust addresses that match the ledger generated address.
```
Comment (obiyankenobi) [inline doc/WALLET_DESKTOP.md:68]: "lose", with just 1 "o".

And split sentences.

File: doc/WALLET_DESKTOP.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+Go to the "Send Tokens" screen and here you can enter which outputs your transaction is going to have.
+
+![](images/05-send-tokens.png)
+
+Output info required:
+
+- Destination address
+- Value
+
+You can add up to 255 outputs for any transaction and time lock is disabled for ledger transactions.
```
Comment (obiyankenobi) [inline doc/WALLET_DESKTOP.md:81]: I'd add the timelock disclaimer in the end, with the disclaimer that only HTR is supported. So 2 disclaimer items.

File: doc/WALLET_DESKTOP.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+Remember: The sum of the outputs values must be equal or less than your available balance.
+
+Now we need to enter from where we will spend the tokens, this can be done automatically in which case the wallet will select the UTXOs that can match the values of the outputs,
+or you can choose inputs manually, in which case you will need to provide for each input:
```
Comment (obiyankenobi) [inline doc/WALLET_DESKTOP.md:85]: The wallet can automatically select the inputs for your transaction. Optionally, you can select them yourself, in which case you need to provide for each input:
- Transaction id;
- Output index (which output from the given transaction will be spent);

File: doc/WALLET_DESKTOP.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+The transaction will not be sent if the inputs total value do not match the outputs total value.
+If the inputs total value exceeds the outputs total value the wallet will generate a change output with the remainder of the tokens.
+(Ledger will verify that the output belongs to your wallet if it exists, so you can focus on confirming the outputs you intend to send)
+
+OBS: Only HTR (Hathor native token) can be sent with the ledger device.
+
+
+Once all inputs and outputs are setup, the wallet will send this information to the Nano S and it will ask for your confirmation.
+
+First you will need to confirm each output, remember to check the address and values.
```
Comment (obiyankenobi) [inline doc/WALLET_DESKTOP.md:99]: Split sentences.

File: doc/WALLET_DESKTOP.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+If all outputs are confirmed, the device will ask your permission to sign the transaction,
```
Comment (obiyankenobi) [inline doc/WALLET_DESKTOP.md:100]: Split sentence in the end of this line.

File: doc/WALLET_DESKTOP.md [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+if confirmed, you will need to wait for the signing process to be done.
+(You can also exit while the signing is ongoing, making the transaction fail)
+
+The wallet will get the sign information and send the transaction, after which you will be directed to the `Wallet` screen.
```
Comment (obiyankenobi) [inline doc/WALLET_DESKTOP.md:104]: will get the **signature** information

**Review by obiyankenobi — CHANGES_REQUESTED** [review]

File: doc/WALLET_DESKTOP.md (hunk 1 above)
Comment (r4mmer) [reply doc/WALLET_DESKTOP.md]: Done

File: doc/WALLET_DESKTOP.md (hunk 2 above)
Comment (r4mmer) [reply doc/WALLET_DESKTOP.md]: Changed the title, but not the file name.

File: doc/WALLET_DESKTOP.md [hunk 13]
```diff
@@ -0,0 +1,104 @@
+# Wallet Desktop
+
+## Overview
+
+We will teach how to connect a ledger Nano S device with hathor wallet and how to operate the main features of the wallet with the ledger device.
```
[the remark this answers is not in the document]
Comment (r4mmer) [reply doc/WALLET_DESKTOP.md]: Done

File: doc/WALLET_DESKTOP.md [hunk 14]
```diff
@@ -0,0 +1,104 @@
...
+
+First the device should be connected to the PC and then we should open the wallet (on the PC) and the hathor app (on Nano S).
+
+![Choose wallet screen](images/01-choose-type.png)
+
+Once you get to the screen above choose `Hardware wallet` and wait with the hathor app open, the wallet will find the device and check if the hathor app is open.
+
+![Second step](images/02-wait-find-device.png)
+
+When the wallet finds the Nano S the second step will begin, the wallet will ask permission to access the wallet public key.
```
[the remark this answers is not in the document]
Comment (r4mmer) [reply doc/WALLET_DESKTOP.md]: Changed

File: doc/WALLET_DESKTOP.md [hunk 15]
```diff
@@ -0,0 +1,104 @@
...
+- [View Public Address](#view-public-address)
+- [Send HTR Tokens](#send-htr-tokens)
+
+## Connect
+
+First the device should be connected to the PC and then we should open the wallet (on the PC) and the hathor app (on Nano S).
+
+![Choose wallet screen](images/01-choose-type.png)
+
+Once you get to the screen above choose `Hardware wallet` and wait with the hathor app open, the wallet will find the device and check if the hathor app is open.
```
[the remark this answers is not in the document]
Comment (r4mmer) [reply doc/WALLET_DESKTOP.md]: Done

File: doc/WALLET_DESKTOP.md [hunk 16]
```diff
@@ -0,0 +1,104 @@
...
+
+![Second step](images/02-wait-find-device.png)
+
+When the wallet finds the Nano S the second step will begin, the wallet will ask permission to access the wallet public key.
+
+To confirm access, go to the `Approve` screen on the Nano S and press with both buttons.
+
+Now the wallet will load your addresses and transaction history, on the next screen you will be ready to send and receive HTR tokens.
```
[the remark this answers is not in the document]
Comment (r4mmer) [reply doc/WALLET_DESKTOP.md]: Done

File: doc/WALLET_DESKTOP.md (hunk 3 above)
Comment (r4mmer) [reply doc/WALLET_DESKTOP.md]: Done

File: doc/WALLET_DESKTOP.md (hunk 4 above)
Comment (r4mmer) [reply doc/WALLET_DESKTOP.md]: Done

File: doc/WALLET_DESKTOP.md (hunk 5 above)
Comment (r4mmer) [reply doc/WALLET_DESKTOP.md]: Done

File: doc/WALLET_DESKTOP.md (hunk 6 above)
Comment (r4mmer) [reply doc/WALLET_DESKTOP.md]: Changed

File: doc/WALLET_DESKTOP.md (hunk 7 above)
Comment (r4mmer) [reply doc/WALLET_DESKTOP.md]: Done

File: doc/WALLET_DESKTOP.md (hunk 8 above)
Comment (r4mmer) [reply doc/WALLET_DESKTOP.md]: Done

File: doc/WALLET_DESKTOP.md (hunk 9 above)
Comment (r4mmer) [reply doc/WALLET_DESKTOP.md]: Done

File: doc/WALLET_DESKTOP.md (hunk 10 above)
Comment (r4mmer) [reply doc/WALLET_DESKTOP.md]: Done

File: doc/WALLET_DESKTOP.md (hunk 11 above)
Comment (r4mmer) [reply doc/WALLET_DESKTOP.md]: Dome

File: doc/WALLET_DESKTOP.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (obiyankenobi) [reply doc/WALLET_DESKTOP.md]: There are still several places with ledger instead of Ledger.

**Review by obiyankenobi — APPROVED** [review]

**Pull request merged**